ARGO/src/TransformComponent.cpp

```diff
@@ -80,4 +80,6 @@ void TransformComponent::moveUp()
 void TransformComponent::moveDown()
 {
 	m_pos.y++;
-}
\ No newline at end of file
+}
```

Comment:
Why is the only change in this new lines at the bottom.
This doesn't change anything mate